### PR TITLE
Show account derivation paths in settings

### PR DESCRIPTION
**Pull request by buberdds**
https://app.clickup.com/t/8694m4gwj

### CODE REVIEW

**Comment by lukaw3d** [comment]
Previews
d5fd4724 oasis-wallet pages dev_account_oasis1qr022vvn4txuugt4cm9fxc59claplp07xgxnynxs(extension375x600 + scrollbar)

d5fd4724 oasis-wallet pages dev_

**Review by lukaw3d — APPROVED** [review]

File: src/app/components/Toolbar/Features/Account/DerivationFormatter.tsx [hunk 1]
```diff
@@ -1,24 +1,71 @@
...
+const iconProps = {
+  size: '20px',
+  color: '#fff',
+}
+
 export const DerivationFormatter = (props: DerivationFormatterProps) => {
-  const { t } = useTranslation()
-  const walletTypes: { [type in WalletType]: string } = {
-    [WalletType.UsbLedger]: t('toolbar.wallets.type.usbLedger', 'USB Ledger'),
-    [WalletType.BleLedger]: t('toolbar.wallets.type.bluetoothLedger', 'BLE Ledger'),
```
Comment (lukaw3d) [inline src/app/components/Toolbar/Features/Account/DerivationFormatter.tsx:15]: Can these translation strings be removed from translation.json

File: src/app/components/Toolbar/Features/Account/ImportableAccount.tsx [hunk 2]
```diff
@@ -18,7 +18,6 @@ export const ImportableAccount = ({
       displayCheckbox={true}
       displayAccountNumber={true}
       displayDerivation={{
-        type: account.type,
```
Comment (lukaw3d) [inline src/app/components/Toolbar/Features/Account/ImportableAccount.tsx:21]: d5fd4724 oasis-wallet pages dev_create-wallet(extension375x600 + scrollbar)
could increase accountsPerPage like https://github.com/oasisprotocol/oasis-wallet-web/pull/1917/files

Comment (buberdds) [reply src/app/components/Toolbar/Features/Account/ImportableAccount.tsx:21]: will keep the current value. for tablet and desktop this view is quite big.

**Pull request merged**